packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -174,17 +167,18 @@ export async function createNewEmptyFluidFile(
 	}/items/root:/${filePath}/${encodedFilename}:/content?@name.conflictBehavior=rename&select=id,name,parentReference`;
 
 	return getWithRetryForTokenRefresh(async (options) => {
-		const storageToken = await getStorageToken(options, "CreateNewFile");
+		const url = initialUrl;
+		const method = "PUT";
+		const authHeader = await getStorageToken(
```

Comment:
nitpick: it is a bit confusing to have authHeader to be returned by getStorageToken method.

Reply:
I thought I renamed all of them. Clearly I didn't! Will fix.